docker/Dockerfile

```diff
@@ -53,6 +53,16 @@ RUN set -e; \
 
 RUN set -e; \
     export DEB_MAX_ATTEMPT=10; \
+    export CRIS_IMG_ARCH=$(case $(uname -m) in \
+        'amd64' | 'x86_64' | 'x64') \
+            echo 'amd64'; \
+            ;; \
+        'arm64' | 'aarch64') \
+            echo 'arm64'; \
+            ;; \
+        *) \
+            echo "unknown"; \
```

Comment:
FYI this won't guarantee non-emptiness even when it's correctly written, as the subshell can still crash.

Reply:
ack